README.md

```diff
@@ -4,92 +4,80 @@ Deploys the Open Cluster Management Search v2 components.
 
 ## Installing the Search Operator in a Red Hat OpenShift cluster
 
-### 0. Prerequisites
```

Comment:
NOTE: The only addition is the **##Development** section. All other changes are formatting.